independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -85,13 +84,20 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                         .noDockerfiles(invocation.getValue(NO_DOCKERFILES, false))
                         .addData(platformData)
                         .addData(LegacySupport.convertFromLegacy(invocation.getValues()))
+                        .messageWriter(invocation.log())
                         .build();
-                invocation.log().info(
-                        "Generating Quarkus Codestart Project with data: " + input.getCodestartInput().getData().toString());
+                invocation.log().info("-----------");
+                invocation.log().info("selected extensions: \n"
+                        + extensionsToAdd.stream().map(e -> "- " + e.getGroupId() + ":" + e.getArtifactId() + "\n")
+                                .collect(Collectors.joining()));
                 final CodestartProject codestartProject = prepareProject(input);
-                invocation.log().info("Codestarts: " + codestartProject.getCodestarts().stream().map(Codestart::getSpec)
-                        .map(CodestartSpec::getName).collect(Collectors.joining(", ")));
                 Codestarts.generateProject(codestartProject, invocation.getQuarkusProject().getProjectDirPath());
+                invocation.log()
+                        .info("\n-----------\n" + MessageIcons.NOOP_ICON + " "
+                                + codestartProject.getRequiredCodestart(CodestartType.PROJECT).getName()
```

Comment:
Minor comment: It would be nice to display the project name in bold

Reply:
Ok we could do that in a another PR